src/bot/dragonfly_services.py

```diff
@@ -157,3 +157,12 @@ async def report_package(
         """Report a package to Dragonfly."""
         data = dataclasses.asdict(report)
         await self.make_request("POST", "/report", json=data)
+
+    async def queue_package(self: Self, name: str, version: str) -> None:
+        """Add a package to the Dragonfly scan queue."""
+        data = {
+            "name": name,
+            "version": version,
+        }
+
+        await self.make_request("POST", "/package", json=data)
```

Comment:
This is fine, but technically is broken on mainframe's side. Currently only the latest version can be queued.

https://github.com/vipyrsec/dragonfly-mainframe/blob/6187f7b460ae89ecc29a65647c9aa6b3bd5ccb27/src/mainframe/endpoints/package.py#L278.

Reply:
Any version can be queued, but this line is redundant:
```py
version = package_metadata.releases[0].version  # Use latest version if not provided
```